.github/workflows/pythonapp.yml

```diff
@@ -0,0 +1,55 @@
+name: Housekeeper tests
+
+on: push
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
+
+    steps:
+
+    # Check out Scout code
+    - uses: actions/checkout@v1
+
+    # Set up python
+    - name: Set up Python ${{ matrix.python-version}}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version}}
+
+    - name: Production Dependencies
+      run: |
+        python -m pip install --upgrade pip
```

Comment:
Sure, I don't have any strong opinion. Although it might be hard to maintain it that way, then we must update all these files in all packages every time we update prod. I would suggest that we always run `python -m pip update -U pip ` when environments are updated

Reply:
What I mean is that it should be a controlled timing to do it like for instance a PR just to update all dependencies to the newest versions and check that all functionality is still ok. But i think you know more than me about piplock. I just want to protect from packages that get updated with breaking changes (if that is the case) automatically. I guess it is easy to revert to an older version and reimplement if it can happen

Reply:
Yes I think this will do for now but it is definitely something to work for. Thats why I'm adding these Pipfiles now. I need more time to motivate and discuss, hope that will be on the development day.